docs/variables/valorant.md

```diff
@@ -0,0 +1,153 @@
+
+:::caution Authorization required!
+
+This variable will only function if a broadcaster has authorized and connected a Riot Games account within Fossabot's dashboard under the **Integrations** tab.
+
+:::
+
+* **Supported Game Modes**
+  * `competitive`
+  * `deathmatch`
+  * `escalation`
+  * `premier`
+  * `replication`
+  * `spikerush`
+  * `swiftplay`
+  * `unrated`
+
+* **Special Modifiers**
+  * `include_custom` *(adds custom games to be considered)*
+  * `custom_only` *(limits the considered games to **custom games only**)*
+
+## $(valorant.stream_wins)
+
+Returns the amount of games won that were played in the specified game mode(s) during the current stream.
+
+#### Parameters
+
+This variable takes **one** or more *optional* parameters that is a list of **game modes** which to return the current stream's amount of wins for. Defaults to all existing game modes, excluding custom games, if not provided.
+
+#### Example Output
+
+* `$(valorant.stream_wins)` *(all games modes, **without** custom games)*
+
+    ```
+    1
+    ```
+
+* `$(valorant.stream_wins include_custom)` *(all games modes, **with** custom games)*
+
+    ```
+    3
+    ```
+
+* `$(valorant.stream_wins custom_only)` *(all games modes, **but only** custom games)*
+
+    ```
+    2
+    ```
+
+* `$(valorant.stream_wins spikerush)` *(only Spike Rush games, **without** custom games)*
+
+    ```
+    0
+    ```
+
+#### Error Output
+
+* In case an invalid game mode is specified, returns the following:
+
+    ```
+    [Error: Queue must be one of: competitive, deathmatch, escalation, premier, replication, spikerush, swiftplay, unrated.]
+    ```
+
+## $(valorant.stream_losses)
+
+Returns the amount of games lost that were played in the specified game mode(s) during the current stream.
+
+#### Parameters
+
+This variable takes **one** or more *optional* parameters that is a list of **game modes** which to return the current stream's amount of losses for. Defaults to all existing game modes, excluding custom games, if not provided.
+
+#### Example Output
+
+* `$(valorant.stream_losses)` *(all games modes, **without** custom games)*
+
+    ```
+    1
+    ```
+
+* `$(valorant.stream_losses include_custom)` *(all games modes, **with** custom games)*
+
+    ```
+    3
+    ```
+
+* `$(valorant.stream_losses custom_only)` *(all games modes, **but only** custom games)*
+
+    ```
+    2
+    ```
+
+* `$(valorant.stream_losses spikerush)` *(only Spike Rush games, **without** custom games)*
+
+    ```
+    0
+    ```
+
+#### Error Output
+
+* In case an invalid game mode is specified, returns the following:
+
+    ```
+    [Error: Queue must be one of: competitive, deathmatch, escalation, premier, replication, spikerush, swiftplay, unrated.]
+    ```
+
+## $(valorant.stream_ties)
+
+Returns the amount of games tied that were played in the specified game mode(s) during the current stream.

```

Comment:
Would this read better as

```
Returns the number of games that ended in a tie for a specified game mode(s) during a given stream session.
```

Reply:
sounds good, but I think we should leave it `the current stream session`